azure-pipelines.yml

```diff
@@ -20,14 +21,20 @@ jobs:
     displayName: 'Test'
   - script: ./helpers/update-site.sh
     displayName: 'Deploy'
-    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    condition: and(succeeded(), in(variables['Build.SourceBranch'], 'refs/heads/master', 'refs/heads/functions/serverless'))
```

Comment:
Is this to test that everything is working? Should we remove it now (or eventually)?

Reply:
I'm going to restore this before merge.